src/visualization/trigram.cc

```diff
@@ -305,16 +312,15 @@ void TrigramWidget::setManipulator(Manipulator* manipulator) {
 
 void TrigramWidget::autoSetBrightness() {
   auto new_brightness = suggestBrightness();
-  if (new_brightness == brightness_) return;
-  brightness_ = new_brightness;
-  if (brightness_slider_ != nullptr) {
```

Comment:
I prefer to not have them in such shorts and simple ifs (IMO: no gain in readability if indentation is correct, but one line longer). I probably won't touch them in future PRs ;)